### ISSUE TITLE
Change the message on texter panel when the campaign is dynamic.

### ISSUE THREAD

**Issue by theochino** [ProgressiveCoders/Spoke#910]
Change the 501 - ??? when the campaign is dynamic.
Something like "Dynamic campaign: Sent XXX" 

![image](https://user-images.githubusercontent.com/5959961/45502208-646e8a00-b751-11e8-897c-bf9f00442e7c.png)

**Comment by theochino** [comment]
Also "Send First Text" should say something else than (1)

![image](https://user-images.githubusercontent.com/5959961/45503795-f11b4700-b755-11e8-8995-13022f3abd99.png)

**Comment by ZevH** [comment]
I agree these are both confusing and often mislead novice texters. An ellipsis might be good for both of these situations, to indicate an going process

For the counter: "501 ..." instead of "501-?"

For the "SEND FIRST TEXTS" button, "⋯" (U+22EF) in a green circle instead of "1".

**Comment by ibrand** [comment]
#1533 will change dynamic assign around a bit so this wouldn't be a problem
